Cpp/include/odin/group.hpp

```diff
@@ -0,0 +1,29 @@
+/**
+    Copyright 2018 Felspar Co Ltd. <http://odin.felspar.com/>
+
+    Distributed under the Boost Software License, Version 1.0.
+    See <http://www.boost.org/LICENSE_1_0.txt>
+*/
+
+
+#pragma once
+
+
+#include <fost/postgres>
+
+
+namespace odin {
+
+
+    namespace group {
```

Comment:
Why do we need this namespace?

Reply:
I think it's easier to read the code that uses it, like `odin::group::add_permission(...)`. What do you think?

Reply:
The problem is that we have groups in two places now don't we? And this is only one of them.